mutiny-core/src/lib.rs

```diff
@@ -456,6 +456,17 @@ impl<S: MutinyStorage> MutinyWallet<S> {
                         .nwc_uri
                 };
 
+                let nwc = match nwc {
+                    Some(nwc) => nwc,
+                    None => {
+                        log_error!(
+                            self.node_manager.logger,
+                            "Create NWC without valid connection string"
+                        );
+                        return Err(MutinyError::NostrError);
+                    }
+                };
+
```

Comment:
Was this happening here? This really shouldn't be a problem on `ensure_mutiny_nwc_profile`. 

In fact, if you're returning an error here, then it might cause the node not to start up since that happens on startup (or might happen on startup soon).